src/pages/currency-trading/forms/TransactionForm.js

```diff
@@ -380,11 +400,37 @@ export default function TransactionForm({ recordId, labels, maxAccess, plantId,
     if (!recordId) {
       formik.setFieldValue('reference', record.headerView.reference)
     } else {
+      setOperationType(record.headerView.functionId)
+
       formik.setValues({
         recordId: recordId,
         reference: record.headerView.reference,
-        rows: record.items,
-        rows2: record.cash,
+        rows: record.items.map(
+          ({ seqNo, currencyId, currencyName, currencyRef, lcAmount, fcAmount, minRate, maxRate, ...rest }) => ({
+            id : seqNo,
+            currencyId: currencyId,
+            currency :{
+             recordId: currencyId, name :currencyName, reference :currencyRef
+            },
+            lcAmount: getFormattedNumber(lcAmount),
```

Comment:
Can't we format that inside the datagrid numberfield component? I don't want this page to know anything about formatting.